webpack.config.babel.js

```diff
@@ -70,12 +70,18 @@ module.exports = {
         new webpack.DefinePlugin({
             BM_RPC_ADDR: JSON.stringify(BM_RPC_ADDR),
         }),
+        new webpack.DefinePlugin({
```

Comment:
You don't need to create another one instance of `webpack.DefinePlugin`. You can define `HELPIK_URL` in the instance above.